docs/docs/api-ref.md

```diff
@@ -310,7 +310,7 @@ Name | Description
 :--- | :--- 
 `datasource_id` |  The identifier (`id`) for the the `DatasourceItem` that you want to download from the server. 
 `filepath` |  (Optional) Downloads the file to the location you specify. If no location is specified (the default is `Filepath=None`), the file is downloaded to the current working directory. 
-
+`no_extract` | (Optional) Specifies whether to download the file without the extract. When the data source has an extract, if you set the parameter `no_extract=True`, the extract is not included. You can use this parameter to improve performance if you are downloading data sources that have large extracts. The default is to include the extract, if present (`no_extract=False`). 
```

Comment:
Good point. We should probably add the "available in REST version XX" for all of these. Or some.